homeassistant/components/google_cloud/tts.py

```diff
@@ -151,6 +159,7 @@ async def async_get_engine(hass, config, discovery_info=None):
         config.get(CONF_PITCH),
         config.get(CONF_GAIN),
         config.get(CONF_PROFILES),
+        config.get(CONF_TEXT_TYPE),
```

Comment:
```suggestion
        config[CONF_TEXT_TYPE],
```

Reply:
Can you please explain why you think this is needed?
`
```suggestion
        config.[CONF_TEXT_TYPE],
```
I just can't get it why it's recommended here but it is not in other places:
```` config.get(CONF_LANG),
        config.get(CONF_GENDER),
        config.get(CONF_VOICE),
        config.get(CONF_ENCODING),
        config.get(CONF_SPEED),
        config.get(CONF_PITCH),
        config.get(CONF_GAIN),
        config.get(CONF_PROFILES),

Reply:
The platform config validation schema will put a default in place so we don't need to and should not use `dict.get`. The old lines that use `dict.get` can be updated too as long as there's a default in the config schema. But that's not required to merge this PR.

Reply:
Note that your suggestion has a misplaced period before the bracket. My suggestion has the correct syntax.

Reply:
Yep! Thanks! Just made new commit with suggested changes. Replaced `.get()` where schema had default values. Waiting for PR to be merged!